CHANGELOG.md

```diff
@@ -11,6 +11,23 @@
 * Add support for pausing reconciliations to the Unidirectional Topic Operator
 * Allow running ZooKeeper and KRaft based Apache Kafka clusters in parallel when the `+UseKRaft` feature gate is enabled
 * Add support for metrics to the Unidirectional Topic Operator
+* The `io.strimzi.kafka.EnvVarConfigProvider` configuration provider is now deprecated. Users should migrate to Kafka's implementation, `org.apache.kafka.common.config.provider.EnvVarConfigProvider`, which is a drop-in replacement. 
+  For example:
+  ```yaml
+  config:
+    # ...
+    config.providers: env
+    config.providers.env.class: io.strimzi.kafka.EnvVarConfigProvider
+    # ...
+  ```
+  becomes
+  ```yaml
+  config:
+    # ...
+    config.providers: env
+    config.providers.env.class: org.apache.kafka.common.config.provider.EnvVarConfigProvider
+    # ...
+  ```
```

Comment:
You should add that the Strimzi provider will work until Strimzi 0.42 and then it will be removed as that is in the proposal.

Reply:
Yes that's a good idea to include that information here. I've pushed an update doing so.